Nrefrigetor/Nrefrigetor/Shelf.cs

```diff
@@ -0,0 +1,175 @@
+        public int  FloorNum { get;   private set; }
+        public void setFloorNUm(int numF,Refrigerator r)
+        {
+            if (numF > 0)
+                if (numF == r.getShelvesList().Count) 
+                {
+                    this.FloorNum = numF;
+                }
+        }
+
+        public double ShelfSizeOnSMR { get { return shelfSizeOnSMR; } set { if (value > 0) shelfSizeOnSMR = value; } }
+        public Shelf(double SMR)
+        {
+            this.shelfId = IdGenrator.giveId(); 
+            this.ShelfSizeOnSMR = SMR;
+           
+        }
+        public List<Item> getMyList()
+        {
+            return items;
+        }
+
+        public bool addItem(Item item)
+        {
+            if (this.placewasLeft() - item.PlaceOnSMR >= 0)
+            {
+                item.SetFloorNum(this.floorNum, this);
+                items.Add(item);
+                return true;
+            }
+            else
+            {
+                Console.WriteLine("we dont have place for this item");
+                return false;
+            }
+        }
+
+        public override string ToString()
+        {
+            string sn;
+            if (this.FloorNum == 0)
+                sn = "no floornum yet";
+            else
+            {
+                sn ="the floornum is "+ this.FloorNum.ToString();
+            }
+          
+            if (items!=null)
+                {
+                string itemsToSring = @"    the item detelies is  :";
+                foreach (Item item in items)
+                {
+                    itemsToSring += item.ToString() + ",";
+                }
+               
+               
+                return @" shelfid is " + this.shelfId  +" "+
+                    sn+ 
+                    "  the shelfOnSmr is "
+                    +this.shelfSizeOnSMR + " " +
+                    itemsToSring;
+                }
+            else
+              return
+                   "shelfid " +this.shelfId + " " + sn +"shelfOnSmr" +this.shelfSizeOnSMR;
+        }
+       
+        public double placewasLeft()
+        {
+            if (items==null)
+            {
+                return ShelfSizeOnSMR;
+            }
+            double places = 0;
+            foreach (Item item in items)
+            {
+                places += item.PlaceOnSMR;
+            }
+            return this.shelfSizeOnSMR - places;
+        }
+
+      
+        
+        public Item takeOutItem(int itemid)
+        {    if(items==null)
+            { return null; }
+            foreach (Item item in items)
+            {
+                if (item.ItemId == itemid)
+                {
+                    items.Remove(item);
+                    Console.WriteLine("we remove the item from floor:/n {0}", this.FloorNum);
+                    return item;
+                }
+
+            }
+
+            return null;
+        }
+
+        public void itemAreExpired()
```

Comment:
Naming: Function name needs to be a command, and describe shortly the action taken